src/app/containers/ChartbeatAnalytics/canonical/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,3 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`CanonicalChartbeatAnalytics should return script snippet 1`] = `null`;
```

Comment:
Is this snapshot correct?

Reply:
yeah, apparently that's what it returns which is weird. similar to [this](https://github.com/bbc/simorgh/blob/latest/src/app/components/AudioVideoHead/__snapshots__/index.test.jsx.snap#L3)